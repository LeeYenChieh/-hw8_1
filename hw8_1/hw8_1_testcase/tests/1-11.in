i46xwjljf2dieJ4M1M oM0Swb5RULaC3oFmq3tpv3VsTY7tD9pIGMgqdn151a2R0WnWoUwSq
i46xwjljf2dieJ4M1M B2UHv4OWNaZ3GWEjmhvbuG
i46xwjljf2dieJ4M1M display
i46xwjljf2dieJ4M1M add scrollBar
i46xwjljf2dieJ4M1M display
i46xwjljf2dieJ4M1M add thickBlackBorder thickBlackBorder thickBlackBorder
i46xwjljf2dieJ4M1M display
i46xwjljf2dieJ4M1M display
i46xwjljf2dieJ4M1M display
i46xwjljf2dieJ4M1M display
i46xwjljf2dieJ4M1M add 