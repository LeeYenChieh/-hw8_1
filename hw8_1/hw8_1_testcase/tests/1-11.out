B2UHv4OWNaZ3GWEjmhvbuG 
B2UHv4OWNaZ3GWEjmhvbuG scrollBar 
B2UHv4OWNaZ3GWEjmhvbuG scrollBar thickBlackBorder thickBlackBorder thickBlackBorder 
B2UHv4OWNaZ3GWEjmhvbuG scrollBar thickBlackBorder thickBlackBorder thickBlackBorder 
B2UHv4OWNaZ3GWEjmhvbuG scrollBar thickBlackBorder thickBlackBorder thickBlackBorder 
B2UHv4OWNaZ3GWEjmhvbuG scrollBar thickBlackBorder thickBlackBorder thickBlackBorder 
